tests/common/helpers/bgp.py

```diff
@@ -115,7 +115,8 @@ def teardown_session(self):
         if not self.is_passive:
             for asichost in self.duthost.asics:
                 logging.debug("update CONFIG_DB admin_status to down")
-                asichost.run_sonic_db_cli_cmd("CONFIG_DB hset 'BGP_NEIGHBOR|{}' admin_status down".format(self.ip))
+                if asichost.namespace == self.namespace:
```

Comment:
Can we move the above `logging.debug(...)` under this if statement as well, please? Maybe add the asic namespace in the this log too, e.g. "update CONFIG_DB admin_status to down on {namespace}". Thanks!

Reply:
Hi @cyw233, updated the logging as suggested. Please take a look.